modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -1,32 +1,37 @@
-= Add Behavioral Headers to Simulated API Calls
+= Behavioral Headers for Simulated Calls to REST Methods
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 :imagesdir: ../assets/images
 
-You can use the following behavioral headers to change the behavior of the mocking service. Use behavioral headers where you would normally specify other types of header when simulating requests to API specifications.
+You can use the following headers to change the behavior of the mocking service.
 
+== Introduce Delays into Response Times
+
+Use the `MS2-Delay` header to indicate how many milliseconds the request should take to answer. You might want to use this header to simulate load on your server or latency in a network.
 
-MS2-Delay:: Indicates how many milliseconds the request should take to answer. You might want to use this header to simulate load on your server or latency in a network.
-+
 You can specify a value for this header as an integer or as a range of integers. If you specify a range, the value must have the form `min-max`. For example, given the value `500-1000`, the request will take a random time between 500 and 1000 milliseconds.
-+
+
 [NOTE]
 ====
 It is possible for the delay to be longer than a single integer value or the high end of a range, depending on the latency of your network.
 ====
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
+== Restrict the Range of Error Status Codes That Can Be Returned
```

Comment:
Similar to above:

> Restrict Status Codes

Reply:
Same comment as above. Also, your heading is less clear.